dbfit-java/core/src/test/java/dbfit/util/DbParameterAccessorsMapBuilderTest.java

```diff
@@ -7,10 +8,12 @@
 import static org.hamcrest.Matchers.*;
 
 import java.util.Map;
+import java.util.HashMap;
```

Comment:
Hmm, I don't see that `new HashMap<>()`... Maybe GitHub shows a different version. I'll fetch locally to check again.